meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8qm-mek/bootloader.uuu.in

```diff
@@ -8,20 +8,21 @@ CFG: FB: -vid 0x0525 -pid 0x4030
 CFG: FB: -vid 0x0525 -pid 0x4031
 
 SDPU: delay 1000
-SDPU: write -f imx-boot-mfgtool -offset 0x57c00
+SDPU: write -f u-boot-mfgtool.itb
 SDPU: jump
```

Comment:
Do we need both SDPU and SDPV parts? I might be wrong, but it seems all imx8 variants support SDPV.
It might be out of the scope of this PR, but I really curious about that trick.

Reply:
it's more universal, as we can use the same script for SPL and non-SPL builds

Reply:
there is a comment below about this:
```
# These commands will be run when use SPL and will be skipped if no spl
# if (SPL support SDPV)
# {
```